eventuate-tram-view-support-rebuild/src/main/java/io/eventuate/tram/viewsupport/rebuild/DomainSnapshotExportService.java

```diff
@@ -169,4 +206,9 @@ private void publishDomainEntity(T domainEntity) {
             domainEventWithEntityId.getEntityId().toString(),
             JSonMapper.toJson(message));
   }
+
+  private String generateAnchorMessageId() {
+    return eventuateCommonJdbcOperations.insertIntoMessageTable(idGenerator,
+              "", "", Collections.emptyMap(), eventuateSchema, true);
```

Comment:
It's confusing to have `"", ""` for payload and destination. At least give them names to make it readable.

Also, having a boolean `published` parameter is a bit confusing. It might be clearer to have a method `insertPublishedMessageIntoMessageTable()`

Reply:
ok